app/components/cms_question_and_answer_component/cms_question_and_answer_component.scss

```diff
@@ -30,7 +30,7 @@ $avatar_size: 32px;
       height: $avatar_size;
       width: $avatar_size;
       left: -$avatar_space;
-      bottom: 4px;
+      bottom: 24px;
```

Comment:
Noticed the question avatar has dropped down the website (its due to changing the class on the question to get the correct font) so snuck this fix in here